[2024-09-04T18:08:06.837+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-04T18:08:06.847+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_v3.second_task manual__2024-09-04T18:08:03.821848+00:00 [queued]>
[2024-09-04T18:08:06.853+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_v3.second_task manual__2024-09-04T18:08:03.821848+00:00 [queued]>
[2024-09-04T18:08:06.854+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-04T18:08:06.863+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): second_task> on 2024-09-04 18:08:03.821848+00:00
[2024-09-04T18:08:06.871+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v3', 'second_task', 'manual__2024-09-04T18:08:03.821848+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp9f5toxpv']
[2024-09-04T18:08:06.875+0000] {standard_task_runner.py:105} INFO - Job 93: Subtask second_task
[2024-09-04T18:08:06.876+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=46090) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-04T18:08:06.878+0000] {standard_task_runner.py:72} INFO - Started process 46092 to run task
[2024-09-04T18:08:06.935+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag_v3.second_task manual__2024-09-04T18:08:03.821848+00:00 [running]> on host cedbab6b7256
[2024-09-04T18:08:07.003+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abhay' AIRFLOW_CTX_DAG_ID='my_first_dag_v3' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-04T18:08:03.821848+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-04T18:08:03.821848+00:00'
[2024-09-04T18:08:07.004+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-04T18:08:07.017+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-04T18:08:07.019+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo I am the second task!']
[2024-09-04T18:08:07.039+0000] {subprocess.py:86} INFO - Output:
[2024-09-04T18:08:07.042+0000] {subprocess.py:93} INFO - I am the second task!
[2024-09-04T18:08:07.044+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-04T18:08:07.067+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-04T18:08:07.069+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v3, task_id=second_task, run_id=manual__2024-09-04T18:08:03.821848+00:00, execution_date=20240904T180803, start_date=20240904T180806, end_date=20240904T180807
[2024-09-04T18:08:07.095+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-04T18:08:07.109+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-04T18:08:07.111+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
